flink-datastream-api/src/main/java/org/apache/flink/datastream/api/extension/eventtime/function/OneInputEventTimeStreamProcessFunction.java

```diff
@@ -43,5 +43,6 @@ default void onEventTimeWatermark(
      * Invoked when an event-time timer fires. Note that it is only used in {@link
      * KeyedPartitionStream}.
      */
-    default void onEventTimer(long timestamp, Collector<OUT> output, PartitionedContext<OUT> ctx) {}
+    default void onEventTimer(long timestamp, Collector<OUT> output, PartitionedContext<OUT> ctx)
+            throws Exception {}
```

Comment:
I suggest adding javadoc for the parameters and Exception. 
nit: I am curious why throws Exception is on these methods rather than just catching a throwable in some higher level code?  
Maybe some words like this method should be overridden ....